packages/hierarchy-browser/src/HierarchyBrowser.tsx

```diff
@@ -34,8 +34,9 @@ export const HierarchyBrowser = memo(function HierarchyBrowser({
 	neighbors,
 	settings,
 }: IHierarchyBrowserProps) {
-	const [hierachyDataProvider] = useState<HierarchyDataProvider>(
+	const hierachyDataProvider = useMemo<HierarchyDataProvider>(
 		() => new HierarchyDataProvider(communities, entities, neighbors),
+		[],
```

Comment:
can you add a comment inside the deps array that we're ignoring `communities, entities and neighbors` intentionally?

Reply:
okay do you want me to add a tslint ignore?

Reply:
if it's popping a linter error, then yes. But even if not, I think a comment inside the dep array that it was left intentionally empty will help future readers understand the decision

Reply:
It is only a warning in the linter so it doesn't effect the build but I can add it.

Reply:
The linter warnings are worth working through and resolving. Some of them may not be worth enabling by default. The ones about hooks dependencies are definitely worth taking a close look at

Reply:
I just disabled the rule for the block of code here for both dataproviders. I think that there maybe other packages that have a similar issue for your awareness in case  we decide to do clean up on those later linting errors later.